src/calendar/day/custom/index.js

```diff
@@ -40,11 +53,17 @@ class Day extends Component {
     return shouldUpdate(this.props, nextProps, ['state', 'children', 'marking', 'onPress', 'onLongPress']);
   }
 
+  renderCustomIcon(){
```

Comment:
We want to move this repo to ES6.
Please use arrow functions and remove `.bind`.

Also, please do use curly braces on inline `if`. it's more readable.
please consider destruction (apply to all your PR)
e.g:
```
 const {marking: {withCustomIcon, iconComponent}} = this.props;
```